service/service_construction.go

```diff
@@ -577,6 +633,8 @@ func (s ConstructionService) ConstructionPayloads(
 
 	if isUnwrapRequest(req.Metadata) {
 		tx, unsignedTx, checkFrom, wrappedErr = s.createUnwrapPayload(req)
+	} else if isGenericContractCall(req.Metadata) {
+		tx, unsignedTx, checkFrom, wrappedErr = s.createGenericContractCallPayload(req)
```

Comment:
To my understanding, the `unwrap` is just another ordinary contract call so ideally, we may want to consolidate both into one. But in my opinion, I suggest to tackle that in a separate PR as the current rosetta operation definition for `unwrap` has major distinction(it has only 1 operation in the `ConstructionPreprocessRequest` rather than 2).